examples/fair_prep_results.py

```diff
@@ -0,0 +1,94 @@
+import os
+import sys
+import shutil
+import warnings
+warnings.simplefilter('ignore')
+
+from helper import extract_info
+
+sys.path.append(os.getcwd())
```

Comment:
I thought we don't need this if we run the examples from the root folder?

Reply:
We need it for running from the root directory. 
When we try to run a file in the examples folder from the root, sys.path stores the path of the examples directory but does not store the path of the root. So we need to add the path of root to the path list.